examples/dogfooding/src/index.ts

```diff
@@ -0,0 +1,194 @@
+    const sequenceTotal = numMessages;
+    let sequenceIndex = 0;
+
+    const sendMessage = async () => {
+      try {
+        const message = ProtoSequencedMessage.create({
+          hash: sequenceHash,
+          total: sequenceTotal,
+          index: sequenceIndex,
+          sender: peerId,
+        });
+        const payload = ProtoSequencedMessage.encode(message).finish();
+        const result = await node.lightPush.send(encoder, {
+          payload,
+          timestamp: new Date(),
+        });
+        console.log("light push successes: ", result.successes.length);
+        console.log("light push failures: ", result.failures.length);
+        if (result.successes.length > 0) {
+          // Push to telemetry client
+          telemetryClient.push<TelemetryPushFilter>([
+            {
+              messageType: TelemetryType.LIGHT_PUSH_FILTER,
+              timestamp: Math.floor(new Date().getTime() / 1000),
+              peerIdSender: peerId,
+              peerIdReporter: peerId,
+              sequenceHash: sequenceHash,
+              sequenceTotal: sequenceTotal,
+              sequenceIndex: sequenceIndex,
+              contentTopic: DEFAULT_CONTENT_TOPIC,
+              pubsubTopic: DefaultPubsubTopic,
+            },
+          ]);
+
+          // Update ui
+          const messageElement = document.createElement("div");
+          const messagesSent = document.getElementById("messagesSent");
+          messageElement.textContent = `Message: ${sequenceHash} ${sequenceIndex} of ${sequenceTotal}`;
+          messagesSent.insertBefore(messageElement, messagesSent.firstChild);
+          messagesSent.insertBefore(
+            document.createElement("br"),
+            messagesSent.firstChild
+          );
+
+          document.dispatchEvent(messageSentEvent);
+
+          // Increment sequence
+          sequenceIndex++;
+        }
+        if (result.failures.length > 0) {
+          console.error("Failed to send message", result.failures);
+        }
+        if (sequenceIndex < sequenceTotal) {
+          setTimeout(sendMessage, period); // Schedule the next send
+        } else {
+          document.dispatchEvent(sequenceCompletedEvent);
+        }
+      } catch (error) {
+        console.error("Error sending message", error);
+      }
+    };
+
+    sendMessage(); // Start the recursive sending
+  };
+
+  const startFilterSubscription = async () => {
+    const decoder = createDecoder(DEFAULT_CONTENT_TOPIC);
+
+    const messagesReceived = document.getElementById("messagesReceived");
+    const subscriptionCallback = (message: DecodedMessage) => {
+      const sequencedMessage: any = ProtoSequencedMessage.decode(
+        message.payload
+      );
+
+      // Don't bother reporting messages sent by this same node
+      if (sequencedMessage.sender === peerId) {
+        return;
+      }
+      telemetryClient.push<TelemetryPushFilter>([
+        {
+          messageType: TelemetryType.LIGHT_PUSH_FILTER,
```

Comment:
hm. is it `lightPush` or `filter`? 
if first - I read it as `sent by lightPush`
if second - `received by filter`

Reply:
It's both. If Sender and Reporter are the same, it's considered sent via light push. If different, then received via filter.

Reply:
so, we can avoid this altogether?